test/index.js

```diff
@@ -125,6 +125,42 @@ test('login fallback to couch', t => {
   })
 })
 
+test('login fallback to couch when web login fails cancels opener promise', t => {
```

Comment:
I think there's some typo in this test name - `web login fails cancels opened promise`

Reply:
`opener` is intended here, it is the name of variable which holds the promise that awaits user to press Enter.. 😄